src/mono/mono/metadata/icall-def-netcore.h

```diff
@@ -67,6 +67,7 @@ NOHANDLES(ICALL(DEBUGR_1, "IsAttached_internal", ves_icall_System_Diagnostics_De
 NOHANDLES(ICALL(DEBUGR_2, "IsLogging", ves_icall_System_Diagnostics_Debugger_IsLogging))
 NOHANDLES(ICALL(DEBUGR_3, "Log_icall", ves_icall_System_Diagnostics_Debugger_Log))
 
+#ifndef DISABLE_EVENTPIPE
```

Comment:
Managed EventSource/EventListener is depending on the icalls, so if we want to keep the managed code enabled, we would need to include icalls, but have stub implementation in case where we disable eventpipe library.